templates/cloud_vpn/provisioners/aws_igw/initiator/provision.j2

```diff
@@ -5,7 +5,7 @@ Resources:
     Properties:
       Tags:
         - Key: Name
-          Value: {{ cloud_vpn_name }}-initiator-internetgw
+          Value: {{ cloud_vpn_initiator_internetgateway_name }}
```

Comment:
@victorock +1 on making this flexible 👍 
but why not add a `defaults` var, set to the original value, like:
```
cloud_vpn_initiator_internetgateway_name: {{ cloud_vpn_name }}-initiator-internetgw
```
(or is the var already defined, and I could not find it?)
ps: Same comment applies to the other changes